rustls/src/webpki/verify.rs

```diff
@@ -437,6 +437,18 @@ impl fmt::Debug for WebPkiSupportedAlgorithms {
     }
 }
 
+/// wrapper around internal representation of a parsed certificate. This is used in order to avoid parsing twice when specifying custom verification
```

Comment:
not changed by this commit, but should start with a capital letter, start with a "headline" sentence, and be manually wrapped

```suggestion
/// Wrapper around internal representation of a parsed certificate.
///
/// This is used in order to avoid parsing twice when specifying custom verification.
```

Reply:
Fixed up.